fpmsyncd/fpmsyncd.cpp

```diff
@@ -69,7 +99,14 @@ int main(int argc, char **argv)
                 {
                     warmStartTimer.start();
                     s.addSelectable(&warmStartTimer);
+                    SWSS_LOG_NOTICE("Warm-Restart timer started.");
                 }
+
+                // Also start periodic eoiu check timer, first wait 5 seconds, then check every 1 second
```

Comment:
None of this code should execute unless someone has specifically configured the hack you are proposing even in the warn restart case.

Reply:
@jipanyang Have you addressed this comment?

Reply:
@nikos-github I'd treat this function as integral part of bgp warm restart. If you see any issues for the function itself, I could try to address them.  Prefer not to push some change that is not necessary.

Or you have more optimal solution and implementation to the bgp warm restart route convergence delay problem,  please share your design and code.  We are not in a rush to push any change in this PR,  but thinking this could be useful for organizations who would like to exercise warm-reboot feature.

Reply:
as long as the Azure/sonic-buildimage#2823 is not merged, this eoiu based reconciliation will not be executed. later we can add the mark.py to look for a config db option to enable or disable eoiu based reconciliation.